tests/acceptance/authenticated-user-sees-authenticated-features-test.js

```diff
@@ -24,20 +24,54 @@ module('Acceptance | authenticated user sees authenticated features', function(h
     await invalidateSession();
   });
 
-  hooks.beforeEach(async function() {
+  test('User does not see hearing and recommendation buttons on show-project if logged out', async function(assert) {
+    await visit('/projects/1');
+
+    assert.notOk(find('[data-test-hearing-rec-shortcuts]'));
+  });
+
+  test('User sees hearing and recommendation buttons on show-project if logged in and project assigned to them', async function(assert) {
+    // user has to be signed in and assigned to that project (dcp_name matches)
+    const currentProject = server.create('project', {
+      id: 1,
+      dcp_name: 'P2012M046',
+    });
+
     this.server.create('project', {
       id: 1,
```

Comment:
Looks like there are two projects created, but only one is used?

Reply:
Hmmm I should change the naming. One is for setting the project on the server which represents the current project page. The other one (the `const`) is for setting the `projects` property on the `user`, so that we can set `dcp_name` to whatever we need -- whether the user IS assigned to a project or a user IS NOT assigned to a project (i.e. `dcp_name` matches)

Reply:
Naming changed

Reply:
Oh gotcha, I see 👍